src/main/java/com/github/invader/agent/config/AgentConfiguration.java

```diff
@@ -4,13 +4,18 @@
 
 @Data
 public class AgentConfiguration {
-    private String server;
+    private Config config;
     private String group;
     private String name;
     private Log log = new Log();
     private Bindings bindings = new Bindings();
 
     @Data
+    public static class Config {
+        private String source;
+    }
```

Comment:
Add and handle an interval field, so that the polling interval is also configurable.